net/esniper/Makefile

```diff
@@ -19,7 +19,7 @@ VERSION_TRANSFORMED:=$(subst .,-,$(PKG_VERSION))
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(VERSION_TRANSFORMED)
 PKG_SOURCE:=$(PKG_NAME)-$(VERSION_TRANSFORMED).tgz
 PKG_SOURCE_URL:=@SF/$(PKG_NAME)
-PKG_MD5SUM:=4b411588c4a223acef7333ab684c7618
+PKG_SHA256SUM:=b32e93ee964489d560d9123ce5f87c5cbf94d9a0198196095a0b85411ed4ad6657f7a2531e24c1e51644562b433e418faa56b62abd8f0b6e95ce6014a746d7af
```

Comment:
There is not yet `PKG_SHA256SUM`, you just assign sha256sum value to `PKG_MD5SUM` and the build system will figure out the proper checksum type to use by checking the length, which looks wrong in this diff.